packages/plugin-core/src/FeatureShowcase.ts

```diff
@@ -0,0 +1,78 @@
+import { VSCodeEvents } from "@dendronhq/common-all";
+import { SegmentClient } from "@dendronhq/common-server";
+import { MetadataService, ShowcaseEntry } from "@dendronhq/engine-server";
+import * as vscode from "vscode";
+import {
+  MeetingNoteTestGroups,
+  MEETING_NOTE_FEATURE_SHOWCASE_TEST,
+} from "./abTests";
+import { AnalyticsUtils } from "./utils/analytics";
+
+/**
+ * Class to showcase certain features of Dendron (like a tip of the day). Right
+ * now, we have only 1 feature - we can expand functionality in this class as we
+ * have more tips.
+ */
+export class FeatureShowcase {
+  /**
+   * Show right now uses vscode.showInformationMessage.  Later, we can have
+   * different display locations, such as the proposed Dendron Side Panel
+   */
+  public show(): void {
+    const ABUserGroup = MEETING_NOTE_FEATURE_SHOWCASE_TEST.getUserGroup(
```

Comment:
can we disable this for tutorial workspace? this might be tricky - eg. if this is their first time using dendron and they are on the tutorial, we don't want to show tool tips for new features. maybe limit this to users that have run initialize workspace gt 3 times?

Reply:
Do we keep track of init workspace > 3 times?  

How about instead, we do `firstWsInitialize` at least 1 week ago?